config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root "homes#index"
+  get "/monsters/:id", to: "homes#index"
+
+ get "/api/v1/monsters", to: "api/v1/monsters#index"
 
```

Comment:
This creates two different pattern for our api routes and we will want to pick one to stay consistent.  In this case we should add the `show` route to the namespaced route below for monsters.

Reply:
Done